packages/back-end/src/services/stripe.ts

```diff
@@ -0,0 +1,53 @@
+import { STRIPE_SECRET } from "../util/secrets";
+import { Stripe } from "stripe";
+import { updateOrganizationByStripeId } from "../models/OrganizationModel";
+const stripe = new Stripe(STRIPE_SECRET || "", { apiVersion: "2020-08-27" });
+
+export async function updateSubscription(
+  subscription: string | Stripe.Subscription
+) {
+  // Make sure we have the full subscription object
+  if (typeof subscription === "string") {
+    subscription = await stripe.subscriptions.retrieve(subscription, {
+      expand: ["plan"],
+    });
+  }
+
+  const stripeCustomerId =
+    typeof subscription.customer === "string"
+      ? subscription.customer
+      : subscription.customer.id;
+
+  await updateOrganizationByStripeId(stripeCustomerId, {
+    subscription: {
+      id: subscription.id,
+      qty: subscription.items.data[0].quantity || 1,
+      trialEnd: subscription.trial_end
+        ? new Date(subscription.trial_end * 1000)
+        : null,
+      status: subscription.status,
+    },
+    priceId: subscription.items.data[0].price.id,
+  });
+}
+
+export async function updateStripeSubscription(
```

Comment:
I think the name of these functions is confusing.   Just looking at the names, I would have no idea what the difference is between `updateSubscription` and `updateStripeSubscription` or which one I should use.

Reply:
@jdorn Yea, I struggled with these a bit. I updated them to `updateSubscriptionInDb` and `updateSubscriptionInStripe` (respectively). Curious on feedback on the updated names or if you have suggestions. The each update the subscription, but the first updates it in our DB, whereas the other hits the stripe API.